### PR TITLE
Implemented discontinue medication order functionality

### PR DESCRIPTION
**Pull request by donaldkibet**
image
image

Implemented functionality to discontinue an order.

### CODE REVIEW

**Comment by fatmali** [comment]
Good work Donald. Left a few comments you could review. Also, the save and submit buttons look a little off.

**Comment by jonathandick** [comment]
Let's get conflicts fixed and then we can merge

File: src/widgets/medications/medication-order.component.tsx [hunk 1]
```diff
@@ -66,6 +67,7 @@ export function MedicationOrder(props: MedicationOrderProps) {
         setDoseUnits(response.data.results[0].dosageForm.uuid);
         setDosageForm(response.data.results[0].dosageForm.display);
         setDrugStrength(response.data.results[0].strength);
+        setConcept(response.data.results[0].concept.uuid);
```
[the remark this answers is not in the document]
Comment (donaldkibet) [reply src/widgets/medications/medication-order.component.tsx]: the API returns an array

File: src/widgets/medications/medications.resource.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ export function saveNewDrugOrder(
   abortContoller: AbortController,
   drugOrder: OrderMedication
 ) {
-  if (drugOrder.previousOrder === null) {
+  if (drugOrder.action === "NEW") {
```
[the remark this answers is not in the document]
Comment (donaldkibet) [reply src/widgets/medications/medications.resource.tsx]: :thumbsup:

File: src/widgets/medications/medications.resource.tsx [hunk 3]
```diff
@@ -120,6 +121,26 @@ export function saveNewDrugOrder(
...
       }
     });
+  } else {
+    return openmrsFetch(`/ws/rest/v1/order`, {
+      signal: abortContoller.signal,
+      method: "POST",
+      headers: {
+        "Content-type": "application/json"
+      },
+      body: {
```
[the remark this answers is not in the document]
Comment (donaldkibet) [reply src/widgets/medications/medications.resource.tsx]: some properties on drugOrder don't form part of the body

**Pull request closed without merging**